### DISCUSSION TITLE
Coral Alternative

### DISCUSSION THREAD

**Discussion by scales11** [blakeblackshear/frigate#3311] [General]
Hello all.

I am enjoying Frigate on my home server (i7-2600 3.4GHz) and would like to offload some of the work to a Coral device.  Unfortunately it is extremely difficult to get my hands on one.

Are there any plans to support any alternative device?  I saw a thread about using an NVIDIA gpu but am not sure how effective that would be (in addition to being intimidated by the task).

Any advice would be appreciated, Thanks!

**Comment by NickM-27** [comment]
There are a number of long threads like this, here is an example: https://github.com/blakeblackshear/frigate/discussions/2918

The issue is finding an alternative, there are a number of factors that determine how well a device can handle this and there aren't options that fit it besides what is already being worked on.

https://github.com/blakeblackshear/frigate/pull/2548 adds support for the jetson nano which is also currently pretty expensive / not available at retail. That work does enable support for Nvidia GPU which is quite effective (as long as the GPU has AI cores) but also pretty expensive.

The intel compute NCS2 seemed like a good option but it is already discontinued so no sense in putting the effort to support it if it will be DOA. 

Google has recently contracted ASUS to take up manufacturing the coral so hopefully that will be more available as more are made.

**Comment by NickM-27** [comment]
There's also something to be said about CPUs with built in AI cores like Apple Silicon and new / upcoming intel / AMD CPUs (see some info https://github.com/blakeblackshear/frigate/discussions/3306#discussioncomment-2894386)

**Comment by scales11** [comment]
Thanks for your reply!

Frankly I have major doubts with many of Google's hardware.  In my experience they tend to make something very cool only to abandon it shortly after.

**Reply by NickM-27** [reply]
There's certainly a history of them killing things, given its popularity as well as how applicable it is to their own work I don't think that will be the case here.

Regardless, that fact doesn't really change the fact that there aren't other good options (at least that I know of) besides what has been mentioned.

**Comment by digiltd** [comment]
Still in stock here: https://github.com/blakeblackshear/frigate/discussions/546#discussioncomment-2870170
